Content.Server/_DV/Antag/NukieOperationComponent.cs

```diff
@@ -21,6 +21,13 @@ public sealed partial class NukieOperationComponent : Component
     /// </summary>
     [DataField]
     public ProtoId<NukieOperationPrototype>? ChosenOperation;
+
+
+    /// <summary>
+    ///     Automatically calls war after set time.
+    /// </summary>
+    [DataField]
+    public TimeSpan AutoWarCallTime = TimeSpan.FromMinutes(2);
```

Comment:
put this on its own component `NukieAutoWarComponent` so it doesn't get processed after its been called
then have it only be added to the rule for nuking objective instead of hardcoding a string check